src/Stats.h

```diff
@@ -1,14 +1,15 @@
 #pragma once
 
 #include <QLineEdit>
-#define NUMBER_OF_INT_STATS 2
-#define NUMBER_OF_FLOAT_STATS 1
-#define NUMBER_OF_STATS 4
+#define NUMBER_OF_INT_STATS 6
+#define NUMBER_OF_FLOAT_STATS 3
+#define NUMBER_OF_STATS 10
```

Comment:
This is a bit fragile: whenever we add a new stat, we have to remember to update these constants.

How about using a `QMap<StatsEnum, TYPE>` for all stats, rather than fixed sized arrays?